pydax/_schema.py

```diff
@@ -44,6 +44,10 @@ class Schema(ABC):
     def __init__(self, url_or_path: Union[_typing.PathLike, str], *,
                  tls_verification: Union[bool, _typing.PathLike] = True) -> None:
         """Constructor method.
+
+        Example:
+
+        >>> schema = Schema('./tests/schemata/datasets.yaml')
```

Comment:
I don't think this is supposed to happen. This doesn't error out probably because we don't call `super().__init__(url_or_path=url_or_path, tls_verification=tls_verification)`